worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/clipboard/DiskOptimizedClipboard.java

```diff
@@ -133,7 +133,7 @@ public DiskOptimizedClipboard(BlockVector3 dimensions, File file) {
                 e.printStackTrace();
             }
             this.braf = new RandomAccessFile(file, "rw");
-            long fileLength = (long) getVolume() * 2L + (long) headerSize;
+            long fileLength = (long) getVolume() << 1 + (long) headerSize;
```

Comment:
This is most likely wrong as + has a higher precedence than <<, while + has a lower precedence than *